docs/installation/est/Installing_EST_pkispawn.md

```diff
@@ -0,0 +1,202 @@
+# EST installation using `pkispawn`
+
+After the prerequisite in [Installing
+EST](Installing_EST.md), it is possible to installa **EST**.
+
+## REALM user DB
+
+
+**EST** subsystem has its own realm authentication with a separate
+user DB. Out of the box *LDAP*, *PostreSQL* and file based DB are
+supported. User management is currently not performed by the subsystem
+so the DB has to be prepared in advance.
+
+### LDAP based DB
+
+The user DB requires a node containing the users (*inetOrgPerson*) and
+a node containing the groups (*groupOfUniqueNames*). Therefore, if
+the base dn is like `dc=pki,dc=example,dc=com` it is possible create a
+tree with a user using the command:
+
+```
+ldapadd -x -H ldap://estds.example.com:3389 \
+    -D "cn=Directory Manager"  -w Secret.123 << EOF
+dn: dc=est,dc=pki,dc=example,dc=com
+objectClass: domain
+dc: est 
+          
+dn: ou=people,dc=est,dc=pki,dc=example,dc=com
+ou: people
+objectClass: top
+objectClass: organizationalUnit
+          
+dn: ou=groups,dc=est,dc=pki,dc=example,dc=com
+ou: groups
+objectClass: top
+objectClass: organizationalUnit
+          
+dn: uid=est-test-user,ou=people,dc=est,dc=pki,dc=example,dc=com
+objectClass: top
+objectClass: person
+objectClass: organizationalPerson
+objectClass: inetOrgPerson
+uid: est-test-user
+sn: EST TEST USER
+cn: EST TEST USER
+userPassword: Secret.123
+
+dn: cn=estclient,ou=groups,dc=est,dc=pki,dc=example,dc=com
+objectClass: top
+objectClass: groupOfUniqueNames
+cn: estclient
+uniqueMember: uid=est-test-user,ou=People,dc=est,dc=pki,dc=example,dc=com
+EOF
+```
+
+### PostgreSQL based DB
+
+To use the *PostreSQL* DB the user tables should be created with the sql
+file provided in `/usr/share/pki/est/conf/realm/postgresql/create.sql`
+and then filled with the user information.
+
+It is possible to use different schemas but in this case a custom
+`statements.conf` file (provided in the same folder) has to be
+provided in order to retrieve the user information from the DB.
+
+
+## Installation
```

Comment:
Is this section supposed to be for installing on the same instance, or does it contain something that shared with the installation on separate instance?  It is currently one level above ### Installation on separate instance.

Reply:
The content of this section applies to both cases: same or separate instance. However, in the second case there are a couple of additional steps to get the CA and SSL certificates.